docs/toc.yml

```
@@ -248,6 +248,8 @@ items:
             href: community-toolkit/ollama.md
           - name: Meilisearch
             href: community-toolkit/hosting-meilisearch.md
+          - name: SQL Database Projects
+            href: community-toolkit/hosting-sql-projects.md
```

```suggestion
            href: community-toolkit/hosting-sql-database-projects.md
```
We used this URL in the community toolkit repo readme.